site/styles/_buttons.scss

```diff
@@ -218,3 +218,54 @@ s72-userwishlist-button {
     /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
   }
 }
+
+.s72-btn-sso-veriplay {
+  @extend .btn-primary;
+  background-color: var(--veriplay-button-background);
+  border-color: var(--veriplay-button-background);
+  color: var(--veriplay-button-text-color);
+  width: 100%;
+
+  @include hover() {
+    background-color: var(--veriplay-button-background-hover);
+    border-color: var(--veriplay-button-background-hover);
+    color: var(--veriplay-button-text-color-hover);
+  }
+
+  &:focus,
+  &.focus {
+    border-color: var(--veriplay-button-background-hover);
+    color: var(--veriplay-button-text-color-hover);
+    @if $enable-shadows {
+      @include box-shadow($btn-box-shadow, 0 0 0 $btn-focus-width rgba(var(--primary-rgb), 0.5));
```

Comment:
These primary colours want be changed to the veriplay button background colour too right?

Reply:
:+1:

Reply:
You've marked this as resolved... but this focus style is still using primary colours... should it be?